Material wall/floor and floor tile name/desc changes

cacogen opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

1. Renames material floors to floor instead of plating
2. Removes placeholdery material floor description to make them consistent with regular metal floors which don't have one
3. Changes material wall description to name the material used, to mirror the description of regular metal walls
4. Changes material tiles' name to "floor tile" to be consistent with regular floor tiles
5. Adds custom examine to floor tiles based on their throwforce
5. Buffs material floor tiles' throwforces from 1 to 10 to better showcase the difference between certain materials (e.g. meat, titanium)
5. Radioactive items no longer output a single full stop when examined at a distance
6. Adds warning span to radioactive item message
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Polish, consistency, fixes the mistake of calling material floors plating.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: cacogen
fix: Material floors are now correctly called floors and not plating
balance: Buffs material floor tiles' throwforces from 1 to 10 (same as iron) to better showcase the effect of different materials (e.g. meat vs. titanium)
fix: Radioactive items no longer output a single . when examined at a distance
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

cacogen committed:
Renames material "plating" to floor, changes material wall desc to match regular walls

cacogen committed:
floor tile

cacogen commented:
@Qustinnus i hope you don't mind

cacogen committed:
Custom descs for walls and floor tiles based on material

cacogen committed:
material tile throwforce 1 -> 5

cacogen committed:
switch order of material flags on material tile

Qustinnus commented:
No it's a cool PR I just need to look into this examine thing because I'm not sure this is the right way to do it.

Qustinnus commented on code/game/objects/items/stacks/tiles/tile_types.dm:
```diff
@@ -335,12 +335,21 @@
+	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_UPDATE_DESC | MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS
+
+/obj/item/stack/tile/material/mat_update_desc(mat)
+	switch(throwforce)
```
I think this could be better handled in Examine()

Qustinnus replied:
also I feel like only adding this info to tiles might seem weird.

cacogen replied:
I just wanted to make it consistent with the long-standing current iron floor tile description without misleading players when these tiles are made of a weak material.

Qustinnus replied:
sure, just make sure this is in examine

Qustinnus commented on code/game/turfs/closed/wall/material_walls.dm:
```diff
@@ -17,3 +17,7 @@
 		new M.sheet_type(src, FLOOR(custom_materials[M] / MINERAL_MATERIAL_AMOUNT, 1))
+
+/turf/closed/wall/material/mat_update_desc(mat)
+	desc = "A huge chunk of [mat] used to separate rooms."
```
Mmmh alright I see some more use for this for these walls. However. You should remove the material flag and just run the proc, if the object doesn't update the desc based on materials thats up to them, the proc would just return.

cacogen committed:
Removes material flag and check

- Simplifies floor tile desc throwforce check

cacogen committed:
Moves the throwforce message to examine()

Qustinnus commented on code/game/objects/items/stacks/tiles/tile_types.dm:
```diff
@@ -20,6 +20,11 @@
 
+/obj/item/stack/tile/examine(mob/user)
+	..()
+	if(throwforce >= 10)
```
I feel like there'd be a lack of nuance if at 9 throwforce it already stops being a good throwing weapon, maybe multiple options is better?

cacogen replied:
ok

cacogen committed:
Gives more nuance to tile throwforce examine() messages

cacogen committed:
Simplifies tile examine throwforce code

- Fixes problem with radioactive item examine of outputting a single period when too far away
- Adds spans to tile throwforce examine and radioactive item examine

Rohesie commented on code/datums/components/radioactive.dm:
```diff
@@ -70,9 +70,11 @@
-		else
-			out += "."
-	to_chat(user, out.Join())
+	if(LAZYLEN(out))
+		out += "."
+	else
+		return
```
```suggestion
	if(!LAZYLEN(out))
		return
	out += "."
```
Bit less convoluted logic with an early return.

Rohesie commented on code/game/objects/items/stacks/tiles/tile_types.dm:
```diff
@@ -14,12 +14,32 @@
 	pixel_x = rand(-3, 3)
 	pixel_y = rand(-3, 3) //randomize a little
 
+/obj/item/stack/tile/examine(mob/user)
+	..()
```
Preserve the return and add the extra text to it, instead of adding a second `to_chat()`

cacogen committed:
logic 10

Co-Authored-By: Rohesie <rohesie@gmail.com>

cacogen committed:
Adds throwforce message to examine return
